packages/@aws-cdk/aws-kms/test/integ.key-sharing.lit.expected.json

```diff
@@ -46,7 +46,7 @@
             "Version": "2012-10-17"
           }
         },
-        "DeletionPolicy": "Retain"
+        "DeletionPolicy": "Delete"
       },
```

Comment:
As a good practice it is usually preferable not to modify existing tests but rather add new tests

Reply:
Well... technically I'm only touching here because I want to stop littering the account with one-off keys that we'll never use again... 😅